tools/bumbershoot/macros.xml

```diff
@@ -2,7 +2,7 @@
     <token name="@VERSION@">3.0.10185</token>
     <xml name="requirements">
         <requirements>
-            <requirement type="package" version="3.0.10185">bumbershoot</requirement>
+            <requirement type="package" version="3_0_10185">bumbershoot</requirement>
```

Comment:
DOH! And even with that typo, my toolshed install worked with a clean instance of Jetstream Galaxy... I guess it doesn't require exact version matching. :)